# FunctionalOMICs 

[**FunctionalOMICs**](https://prokischlab.github.io/functionalOMICs/) is an open access online resource created as part of the “Guidelines for clinical interpretation of variant pathogenicity using RNA phenotypes” [study]() by Smirnov et al., led by Dr. Holger Prokisch at the Technical University of Munich. 


According to the [ACMG/AMP guidelines](https://doi.org/10.1038/gim.2015.30), genetic variants with a certainty of pathogenicity greater than 90% should be considered as likely pathogenic. This concept was further extended by [Tavtigian et al. 2018](https://doi.org/10.1038/gim.2017.210) by defining 99% certainty for pathogenic variants and by implementation of ACMG/AMP guidelines as a Bayessian framework. In line with this, [Brnich et al. 2019](https://doi.org/10.1186/s13073-019-0690-2) suggested estimating the magnitude of evidence strength that is appropriate for a given functional assay by calculating the odds of pathogenicity (OddsPath).

Based on the analysis of available data, and the framework developed by [Brnich et al. 2019](https://doi.org/10.1186/s13073-019-0690-2), we propose the best practice in the interpretation of RNA-seq analysis data in the context of ACMG/AMP guidelines. 


### Resource structure
1. **RNA-seq**
  - [Aberrant expression]()
  - [Monoallelic expression]()
  - [Aberrant splicing]()

This repository will be later extended to other functional assays including proteomics, Seahorse, gene-specific enzyme assays.

Contact: <dmitrii.smirnov@helmholtz-muenchen.de> or <prokisch@helmholtz-muenchen.de>.
