<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Dmitrii Smirnov" />

<meta name="date" content="2021-07-09" />

<title>OddsPath for mono-allelic expression</title>

<script src="libR/header-attrs-2.8/header-attrs.js"></script>
<script src="libR/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libR/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libR/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libR/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libR/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libR/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="libR/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="libR/tocify-1.9.1/jquery.tocify.js"></script>
<script src="libR/navigation-1.1/tabsets.js"></script>
<script src="libR/navigation-1.1/codefolding.js"></script>
<link href="libR/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="libR/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="lib/add_content_table.css" type="text/css" />
<link rel="stylesheet" href="lib/leo_style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>



<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">OddsPath for mono-allelic expression</h1>
<h4 class="author">Dmitrii Smirnov</h4>
<h4 class="date">2021-07-09</h4>

</div>


<p>Allele-specific expression analysis was performed with <strong>negative-binomial test</strong> implemented in the <a href="https://github.com/gagneurlab/drop">DROP</a> pipeline.</p>
<ul>
<li>Study: <a href="https://www.medrxiv.org/content/10.1101/2021.04.01.21254633v2.full-text">Yepez et al. 2021</a></li>
<li><strong>Organism:</strong> Homo sapiens</li>
<li><strong>Genome assembly:</strong> hg19</li>
<li><strong>Gene annotation:</strong> gencode34</li>
<li><strong>Tissue:</strong> Fibroblast</li>
<li><strong>Number of samples:</strong> 303</li>
<li><strong>Data:</strong> <a href="https://zenodo.org/record/4646827#.YOcn2hMza3U">Strand specific</a>, <a href="https://zenodo.org/record/4646823#.YOcm9RMza3U">Non strand specific</a></li>
</ul>
<p>Odds of pathogenicity (<strong>OddsPath</strong>), described by <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6336098/">Tavtigian et al., 2018</a> and established for functional assays by <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0690-2">Brnich et al., 2019</a> was calculated as:</p>
<p>OddsPath = (P2<em>(1-P1))/((1-P2)</em>P1),</p>
<p>where <em>P1</em> is prior probability, or proportion of Pathogenic variants in overall data P1 = N_pathogenic/(N_pathogenic + N_benign)</p>
<p>and <em>P2</em> is posterior probability proportion of pathogenic variants in the groups with functionally abnormal or functionally normal readouts P2 = N_pathogenic_aberrant/(N_aberrant+1)</p>
<p><strong>OddsPath</strong> was calculated for different variant types (Coding, Non-coding, PTV and Splice) and for different thresholds : significance or effect size &amp; significance</p>
<ul>
<li>Brnich and colleagues defined evidence strength corresponding to different OddsPath values:</li>
<li>Benign strong (BS3): OddsPath &lt; 0.053</li>
<li>Benign moderate: OddsPath &lt; 0.23</li>
<li>Benign supporting: OddsPath &lt; 0.48</li>
<li>Intermediate: 0.48 &lt; OddsPath &lt; 2.1</li>
<li>Pathogenic supporting: OddsPath &gt; 2.1</li>
<li>Pathogenic moderate: OddsPath &gt; 4.3</li>
<li>Pathogenic strong (PS3): OddsPath &gt; 18.7</li>
<li>Pathogenic very strong: OddsPath &gt; 350</li>
</ul>
<p>Mono-allelic expression, extreme case of allele-specific expression, could be defined by significance &amp; effect size (allele ratio or fold change)</p>
<p>For variants causing MAE, extreme imbalance can serve as evidence of pathogenicity. For mono-allelicly expressed variants no functional evidence could be provided, however, for recessive disorders, allelic evidence of pathogenicity (<em>PM3</em>) could be applied.</p>
<p>Based on <strong>OddsPath</strong> calculations, we investigated how different thresholds affect maximum pathogenicity evidence strength, that could be assigned in the ACMG/AMP framework to a variant causing RNA phenotype</p>
<div id="alternative-allele-ratio-distribution-for-heterozygous-pathogenic-and-benign-variants" class="section level3">
<h3>Alternative allele ratio distribution for heterozygous pathogenic and benign variants</h3>
<pre class="r"><code>p0</code></pre>
<p><img src="Scripts_RNA_seq_4_MAE_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
</div>
<div id="oddspath-for-mae-thresholds" class="section level3">
<h3>OddsPath for MAE thresholds</h3>
<p>based on variants causing MAE ##### panel <strong>a</strong> p-value thresholds ##### panel <strong>b</strong> alternative allele ratio thresholds of nominally significant (p-value &lt; 0.05) MAE</p>
<pre class="r"><code>sup_mae</code></pre>
<p><img src="Scripts_RNA_seq_4_MAE_files/figure-html/unnamed-chunk-7-1.png" width="1248" /></p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
